docs/main/vs-code-ext/plugin-registry.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Plugins
+description: Discovering new Plugins for Capacitor
+contributors:
+  - dtarnawsky
+slug: /vscode/plugins
+---
+
+Plugins provide a Capacitor with great native features, so finding high quality plugins is an important part of implement features in your app.
```

Comment:
"provide a Capacitor with" and "part of implement features" sound a bit off to me.

Reply:
Thanks for checking, i've fixed this.